gen/templates/resource_example_all_attributes.tf.tmpl

```diff
@@ -115,6 +115,12 @@ resource "aci_{{$topContext.ResourceName}}" {{- if ne $formatValue.ContainedBy "
     {{- range $topContext.Properties }}
         {{- if ne .NamedPropertyClass ""}}
   {{overwriteProperty .PkgName .SnakeCaseName $.Definitions}} = aci_{{getResourceName .NamedPropertyClass $.Definitions}}.example.name
+    {{- else if .IgnoreInTest }}
+      {{- if eq .ValueType "bitmask" }}
```

Comment:
Currently in discussion with Shreyas, about this potential change. However, I believe this change in the template is necessary. It is consistent with logic in other parts of this template.
Also, I found that changing:
```
certUsage:
      - "WebSvcOrAuth"
```
to
```
certUsage: ["WebSvcOrAuth"]
```
results in the same value being parsed in the generator. Both yaml formats produce a []interface{}. Hence, there was no difference in the template.

Reply:
@shrsr I have implemented your suggested changes with an additional fix to add `,` between values. This looks good now and the template is simplified a bit more. I like that the logic to format the slice is in the generator instead of the template.
Thank you so much for helping.